src/cascadia/TerminalApp/TerminalPage.h

```diff
@@ -0,0 +1,24 @@
+﻿#pragma once
+
+#include "winrt/Windows.UI.Xaml.Markup.h"
+#include "winrt/Windows.UI.Xaml.Controls.h"
```

Comment:
Sould these Windows.UI ones be in pch.h?

Reply:
I am guessing eventually these be **Microsoft.UI.Xaml** - and using WinUI

Reply:
Possibly they should.

Reply:
Now, they are!